> Tests are failing, can you take a look?

i'm not familiar with `pydantic` so i'm not sure what is wrong here or which side needs to be adjusted

error
```
tests/components/roomba/test_config_flow.py:99: in _mocked_discovery
    roomba = RoombaInfo(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
>   ???
E   pydantic.error_wrappers.ValidationError: 3 validation errors for RoombaInfo
E   sw
E     field required (type=value_error.missing)
E   robotname
E     field required (type=value_error.missing)
E   cap
E     field required (type=value_error.missing)

```

`tests/components/roomba/test_config_flow.py`
```python
def _mocked_discovery(*_):
    roomba_discovery = MagicMock()

    roomba = RoombaInfo(
        hostname="irobot-BLID",
        robot_name="robot_name",
        ip=MOCK_IP,
        mac="mac",
        firmware="firmware",
        sku="sku",
        capabilities={"cap1": 1},
    )
```

`roombapy/roomba_info.py`
```python
class RoombaInfo(BaseModel):
    hostname: str
    firmware: str = Field(alias="sw")
    ip: str
    mac: str
    robot_name: str = Field(alias="robotname")
    sku: str
    capabilities: Dict[str, int] = Field(alias="cap")
    password: Optional[str] = None
```

--- a/tests/components/roomba/test_config_flow.py
+++ b/tests/components/roomba/test_config_flow.py
@@ -98,12 +98,12 @@ def _mocked_discovery(*_):
 
     roomba = RoombaInfo(
         hostname="irobot-BLID",
-        robot_name="robot_name",
+        robotname="robot_name",
         ip=MOCK_IP,
         mac="mac",
-        firmware="firmware",
+        sw="firmware",
         sku="sku",
-        capabilities="capabilities",
+        cap={"cap": 1},
     )
 
     roomba_discovery.get_all = MagicMock(return_value=[roomba])

